tokio-trace/tokio-trace-core/src/callsite.rs

```diff
@@ -67,24 +88,21 @@ pub struct Identifier(
     pub &'static Callsite,
 );
 
+/// Clear and reregister interest on every [`Callsite`]
+///
```

Comment:
It would be great to have a description of when and why this should be used in the docs...